platform-modules/ekstep-framework/src/main/java/org/ekstep/framework/mgr/impl/ChannelManagerImpl.java

```diff
@@ -114,4 +126,26 @@ public Response retireChannel(String channelId) {
 
 		return properties;
 	}
+
+	private void validateLicense(Map<String, Object> request) {
+		if (request.containsKey(ChannelEnum.defaultLicense.name())) {
+			List<Object> licenseList = RedisStoreUtil.getList("license");
```

Comment:
rename Redis key as `edge_license`. 
create a constant variable and use it in all the places.